learn/guides/asyncapi-tool.md

```diff
@@ -43,117 +129,37 @@ The generated Ballerina sources will be written into the same directory from whi
-
-  > **Note:** Currently, this tool supports only HTTP-based event APIs. The path is equal to the header name if the type is `header`, or to the JSON path of the attribute if the type is `body`.
-
-- `x-ballerina-event-type` - this should be there in every event inside the channel. This is the name of the event or the value of the above-mentioned attribute for a specific event.
-
-### Generate Ballerina sources
-
-Execute the command below on the above file. This will generate four files as mentioned above.
-
-```bash
-bal asyncapi -i slack.yaml
-```
-
-### Modify the generated sources
-
-The generated files can be modified according to the custom requirements. When modifying the generated code segments, it will be easier to consider the facts below.
+Then, the generated files can be modified according to the custom requirements. When modifying the generated code segments, it will be easier to consider the below facts.
 
 - All the incoming requests will be coming to the resource function in the `dispatcher_service.bal` file. Hence, if there is a necessity to add an authentication logic for the incoming calls, that logic can be included there before processing the incoming HTTP request.
-- If more information is needed when initializing the listener like secrets, endpoint URLs, tokens, refresh tokens, etc., update the `init` function in the `listener.bal` file.
-
-### Execute the generated sources
+- If more information is needed when initializing the listener like secrets, endpoint URLs, tokens, refresh tokens, and so on, update the `init` function in the `listener.bal` file.
 
-Follow the steps below to use the generated Ballerina sources.
+## Execute the generated sources
 
-1\. Create an empty folder and copy all the generated files to it.
+Follow the steps below to execute the generated Ballerina sources.
 
-2\. Execute the command below from the root of the folder. This generates a `Ballerina.toml` file in the root of the folder.
+1\. Navigate to the directory in which the generated files exist.
+2\. Execute the `bal init` command.
 
-```bash
-bal init
-```
+> **Info:** This will generate a `Ballerina.toml` file.
 
-3\. Create a new Ballerina file inside the folder (e.g., `slack_service.bal`) and paste the code below to it.
+3\. Create a new Ballerina file inside the directory (e.g., `slack_service.bal` ) and copy the code below to it.
 
 ```ballerina
 listener Listener webhookListener = new(8090);
 
 service AppService on webhookListener {
-  remote function onAppMention(GenericEventWrapper event) returns error? {
-      //Implement the logic to use the received `event` here.
-  }
+   remote function onAppMention(GenericEventWrapper event) returns error? {
+       //Implement the logic to use the received `event` here.
+   }
 }
 ```
 
-4\. Execute the command below from the root of the folder to execute this service.
-
-```bash
-bal run
-```
+4\. Execute the`bal run` command to execute this.
```

Comment:
```suggestion
4\. Execute the `bal run` command to execute this.
```

Reply:
Thanks a lot for the suggestion, Resolved